static/app/components/profiling/profilingTransactionHovercard.tsx

```diff
@@ -0,0 +1,301 @@
+    transaction,
+  });
+
+  const triggerLink = <Link to={linkToSummary}>{transaction}</Link>;
+
+  if (!organization.features.includes('profiling-dashboard-redesign')) {
+    return triggerLink;
+  }
+
+  return (
+    <StyledHovercard
+      delay={250}
+      header={
+        <Flex justify="space-between" align="center">
+          <TextTruncate title={transaction}>{transaction}</TextTruncate>
+          <Button to={linkToSummary} size="xs">
+            {t('View Profiles')}
+          </Button>
+        </Flex>
+      }
+      body={
+        <ProfilingTransactionHovercardBody
+          transaction={transaction}
+          project={project}
+          organization={organization}
+        />
+      }
+      showUnderline
+    >
+      {triggerLink}
+    </StyledHovercard>
+  );
+}
+
+function ProfilingTransactionHovercardBody({
+  transaction,
+  project,
+  organization,
+}: ProfilingTransactionHovercardProps) {
+  const {selection} = usePageFilters();
+
+  const baseQueryOptions = {
+    query: `transaction:"${transaction}"`,
+    fields: getProfilesTableFields(project.platform),
+    enabled: Boolean(transaction),
+    limit: 1,
+    referrer: 'api.profiling.profiling-transaction-hovercard',
+    refetchOnMount: false,
+    projects: [project.id],
+  };
+
+  const slowestProfileQuery = useProfileEvents({
+    ...baseQueryOptions,
+    sort: {
+      key: 'profile.duration',
+      order: 'desc',
+    },
+  });
+
+  const latestProfileQuery = useProfileEvents({
+    ...baseQueryOptions,
+    sort: {
+      key: 'timestamp',
+      order: 'desc',
+    },
+  });
+
+  const functions = useFunctions({
+    project,
+    query: '',
+    selection,
+    transaction,
+    sort: '-p99',
+    functionType: 'application',
+  });
+
+  const slowestProfile = slowestProfileQuery?.data?.[0].data[0] ?? null;
+  const durationUnits = slowestProfileQuery.data?.[0].meta.units['profile.duration'];
+  const multiplier = durationUnits ? DURATION_UNITS[durationUnits] ?? 1 : 1;
+
+  const latestProfile = latestProfileQuery?.data?.[0].data[0] ?? null;
+
+  const linkToFlamechartRoute = (
+    profileId: string,
+    query?: {frameName: string; framePackage: string}
+  ) => {
+    return generateProfileFlamechartRouteWithQuery({
+      orgSlug: organization.slug,
+      projectSlug: project.slug,
+      profileId,
+      query,
+    });
+  };
+
+  return (
+    <Flex gap={space(3)} column>
+      <Flex justify="space-between">
+        <ContextDetail
+          title={t('Latest profile')}
+          isLoading={latestProfileQuery.isLoading}
+        >
+          {latestProfile ? (
+            <Link to={linkToFlamechartRoute(String(latestProfile.id))}>
+              {getShortEventId(String(latestProfile!.id))}
+            </Link>
+          ) : (
+            '-'
+          )}
+        </ContextDetail>
+
+        <ContextDetail
+          title={t('Slowest profile')}
+          isLoading={slowestProfileQuery.isLoading}
+        >
+          {slowestProfile ? (
```

Comment:
Should we have some fallback here? Assuming these load our of order or one of them is delayed, the card contents will jump around

Reply:
I took this all into consideration, the loading fallback is handled through the `ContextDetail` component. 

Ontop of that i tried to minimize layout shift as much as possible but... its not perfect 100% but its not janky as you'd expect.

Our `<LoadingIndicator />` is kinda crummy with sizing and doesn't seem to be using `styled` so overriding it to get exact is a pain, we also have a `Placeholder` but it doesnt have a nice loading shimmer effect so instead you just see a grey square.

I'm trying to prevent blocking the whole ui as some queries may take longer than others and the relevant information a user may be looking for may appear faster in some cases.

https://user-images.githubusercontent.com/7349258/212417908-3a7d873b-2d2f-4594-ba3d-8c90696f1803.mov